src/Controller/BatchProcessingController.py

```diff
@@ -191,6 +201,61 @@ def perform_processes(self, interrupt_flag, progress_callback):
                 self.batch_summary[patient]["iso2roi"] = reason
                 progress_callback.emit(("Completed ISO2ROI", 100))
 
+            # Perform SUV2ROI on patient
+            if 'suv2roi' in self.processes:
```

Comment:
I don't think using if... else here to decide the operations to do is maintainable code. Since there will be more values in `self.processes`, the if/else thing makes this `perform_processes` function really long. Can you put the operations for each case in separate functions then call it using a dictionary/class of functions? (There are many suggestions on how to do it here: https://stackoverflow.com/questions/9168340/using-a-dictionary-to-select-function-to-execute)

Reply:
Thank you for this suggestion! It this is okay, it will be left alone for this PR but will be changed in the next one.

Reply:
Sure, as long as they're fixed!

Reply:
Of course, I'm working on it asap in preparation for the next PR.